### PR TITLE
Bug/package updates

### PR DESCRIPTION
**Pull request by leakymemory**
Made a quick pass through the packages and updated most to the current versions.

### CODE REVIEW

File: src/tests/clipperUI/components/previewViewer/previewViewerAugmentationHeader_tests.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ export class PreviewViewerAugmentationHeaderTests extends TestModule {
 				document.getElementById(Constants.Ids.highlightButton).click();
 			});
 
-			let spy = sectionPicker.props.toggleHighlight as Sinon.SinonSpy;
+			let spy = sectionPicker.props.toggleHighlight as sinon.SinonSpy;
```
Comment (VishaalK) [inline src/tests/clipperUI/components/previewViewer/previewViewerAugmentationHeader_tests.tsx:80]: but why would they lower case this :(

Comment (leakymemory) [reply src/tests/clipperUI/components/previewViewer/previewViewerAugmentationHeader_tests.tsx:80]: No idea. https://github.com/DefinitelyTyped/DefinitelyTyped/commit/c0f2401de3e9c207451d271e05f643ed14685ab3

Comment (leakymemory) [reply src/tests/clipperUI/components/previewViewer/previewViewerAugmentationHeader_tests.tsx:80]: Also probably shouldn't have been a minor version bump after this change.

**Comment by VishaalK** [comment]
Build and worked fine on my machine! 

🐑

**Comment by mttwc** [comment]
Confirmed working pairing with @leakymemory

**Pull request merged**